_pages/index.md

```diff
@@ -16,48 +16,104 @@ Login.gov is a standard SAML identity provider, adhering to the [Web Browser SSO
-    </ds:X509Certificate>
-  </ds:X509Data>
-</KeyInfo>
 ```
+**Logout service URL and Binding** - The single logout service URL is used by to contact the Single logout profile. (aka Single Logout Service)
+<br>
+
+**x509 Public Certificate** - The public certificate is used to validate the authenticity of SAML requests received from login.gov. Minimum 2048 bits.
+<br>
+
+### 2. Configuring login.gov to accept your Service Provider
+
+In order to successfully integrate, your Service Provider application details must be whitelisted within login.gov. We require the following details about your service provider application:
+
+`issuer` The Service Provider entity ID
+This is a unique string used to identify your Service Provider with login.gov.
+
 
-In order to successfully authenticate, we will need to whitelist your application. We'll collect the following details about your service provider application:
-
-- `acs_url`
-- `assertion_consumer_logout_service_url`, if supported
-- `sp_initiated_login_url`
-- `block_encryption`
-- `cert`
-- `agency`
-- `friendly name`, optionally
-- `logo (optional)`
-- Return to service provider application URL callback
-- `default attributes`, optionally
+`acs_url` Assertion consumer sign-on URL
+This is your application endpoint which receives authentication assertions.
+
+`assertion_consumer_logout_service_url` Assertion consumer logout service URL
+This is the endpoint which receives logout requests and responses.
+
+`sp_initiated_login_url`* 
+This endpoint initializes authentication with login.gov; it is used to trigger a new authentication request and response at the Service Provider for better usability.
+
+`block_encryption` (optional)
+This defines what type of encryption your Service Provider supports. Currently, only `aes256-cbc` is supported.
+
+`cert` Public certificate
+The public certificate allows login.gov to verify the authenticity of authentication and logout requests.
+
+`agency`*
+This is used to group your Service Provider to an agency, as well as inform the user about what agency the Service Provider belongs.
+
+`friendly name`* (optional)
+This is the user-friendly name for your Service Provider application. 
+
+`logo`* (optional)
+This is your agency or Service Provider's logo. This is used in the header of login.gov when the user is authenticating to your Service Provider. (We will also need the file itself)
+
+`return_to_sp_url`* Return-to Service Provider URL
+This is the URL of the Service Provider which login.gov provides to users when they wish to go directly to the Service Provider site or cancel out of authentication. 
+
+`attribute_bundle`* (optional) The preset bundle of attributes your Service Provider requires
+We allow you to pre-define what attributes your Service Provider requires, or you may request the attributes at run-time.
+
+`*` - Denotes an attribute not found in the SAML WebSSO profile.
+
+#### Sample configuration required by login.gov:
```

Comment:
It is public in `service_providers.yml` but I think linking to the entire file is confusing (especially since `issuer` is not included as a key). Since this is a manual process, I think this is sufficient.